cas-server-core-api-authentication/src/main/java/org/jasig/cas/authentication/RootCasException.java

```diff
@@ -12,7 +12,7 @@
  * see org.jasig.cas.authentication.handler.AuthenticationException
  * @since 4.0.0
  */
-public abstract class RootCasException extends Exception {
+public abstract class RootCasException extends RuntimeException {
```

Comment:
side note, sometimes I wish `RuntimeException` was an interface you could apply to arbitrary Exceptions so you didn't have to choose between making something that should be a `RuntimeException` part of a checked Exception hierarchy that logically makes sense vs. orphaning the exception outside of the checked exception hierarchy so it can stay an unchecked exception.